shell/platform/darwin/ios/framework/Source/FlutterViewController.mm

```diff
@@ -25,6 +25,9 @@
 
 NSNotificationName const FlutterSemanticsUpdateNotification = @"FlutterSemanticsUpdate";
 
+// This is left a FlutterBinaryMessenger privately for now to give people a chance to notice the
+// change.  Unfortunately unless you have Werror turned on, incompatible pointers as arguments
```

Comment:
What's with the double spaces? :)

Reply:
Oh man, that's going to be a hard one for me not to do.  I removed it even though it kills me.